packages/ses/src/compartment-shim.js

```diff
@@ -170,9 +200,19 @@ export class Compartment {
     } = options;
     const localTransforms = [...transforms];
 
-    const { globalTransforms, globalObject } = privateFields.get(this);
+    const {
+      globalTransforms,
+      globalObject,
+      globalLexicals,
+    } = privateFields.get(this);
     const realmRec = getCurrentRealmRec();
-    return performEval(realmRec, source, globalObject, endowments, {
+
+    // TODO just pass globalLexicals as globalObject
```

Comment:
I do not understand this TODO comment

Reply:
In https://github.com/Agoric/SES-shim/pull/368 I remove the `endowments` option. In that world, `globalLexicals` can be passed directly as the `localObject` instead of merging them into an intermediate.

Reply:
So do you mean
```
// TODO just pass globalLexicals as localObject
```
?

Reply:
This comment still seems confused. Do you mean `localObject`?